app/models/user.py

```diff
@@ -0,0 +1,59 @@
+from app.models import *
+
+class User(Base, UserMixin):
+    __tablename__ = "user"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    username = Column(String(20), nullable=False)
+    name = Column(String(20), nullable=False)
+    lastname = Column(String(40), nullable=False)
+    email = Column(String(120), nullable=False)
+    image_file = Column(String(240), nullable=False, default="default.jpg")
+    password = Column(String(120), nullable=False)
+    location = Column(Geography(geometry_type='POINT', srid=4326))
+    domicile_geolocation = Column(Geography(geometry_type='POINT', srid=4326))
+    messages = relationship("Messages", backref="author", lazy=True)
+    profile = relationship("Profile", uselist=False, back_populates="user")
+    user_criteria = relationship("UserCriteria", uselist=False, back_populates="user")
+
+    def __repr__(self):
+        return f"User('{self.username}', '{self.name}', '{self.lastname}','{self.email}', '{self.password}', '{self.image_file}', '{self.location}', '{self.domicile_geolocation}')"
+
+
+
+class Profile(Base, UserMixin):
+    __tablename__ = "profile"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    age = Column(Integer, nullable=False)
+    gender = Column(Enum("male", "female", "custom", name="Gender_Profile"), nullable=False)
+    profile_description = Column(String(500), nullable=False)
+    domicile = Column(String(80), nullable=False)
+    education = Column(String(120), nullable=False)
+    employment_status = Column(Enum("hired", "unemployed", "student", name="Employment_status_Profile"), nullable=False)
+
+    user_id = Column(Integer, ForeignKey("user.id"), unique=True)
+    user = relationship("User", back_populates="profile")
+
+    def __repr__(self):
+        return f"Profile('{self.user}', '{self.age}', '{self.gender}', '{self.profile_description}', '{self.domicile}', '{self.education}', '{self.employment_status}')"
+
+
+class UserCriteria(Base, UserMixin):

```

Comment:
UserMixin should be only used for the user model.
If you take a look at its source code, you'll see it just implement default methods like is_authenticated etc..